examples/enhance/multiscale_gaussian_normalization.py

```diff
@@ -0,0 +1,33 @@
+"""
```

Comment:
Move the example file one folder higher.
For now just put them in `examples/` and no subfolders.